desk/app/groups.hoon

```diff
@@ -3377,7 +3388,7 @@
           seat(roles (~(dif in roles.seat) roles.u-seat))
         (~(put by seats) ship seat)
       ?:  !=(was-admin (go-is-admin our.bowl))
-        (go-restart-updates ~)
+        go-lost-admin
```

Comment:
A little bit implicit here that this will always be the "stopped being admin" case. The branch logic would also produce true for "started being admin", but we're in the `%del-role` branch, which would only ever result in stopping being admin.

Could maybe stand to have a little comment clarifying as such, but fine either way, just something to be cognizant of.